src/adapter/src/coord/sequencer/inner/create_materialized_view.rs

```diff
@@ -657,16 +647,19 @@ impl Coordinator {
 
         let transact_result = self
             .catalog_transact_with_side_effects(Some(session), ops, |coord| async {
+                let output_desc = global_lir_plan.desc().clone();
+                let (mut df_desc, df_meta) = global_lir_plan.unapply();
+
+                df_desc.time_dependence =
+                    time_dependence(coord.catalog(), df_desc.import_ids(), refresh_schedule);
```

Comment:
Could these calls to `time_dependence` be moved into the final `optimize` call, which returns this `df_desc`? Then this call wouldn't need to be duplicated between bootstrapping and sequencing.